apps/api/v1/pages/api/event-types/_utils/checkParentEventOwnership.ts

```diff
@@ -44,7 +40,8 @@ export default async function checkParentEventOwnership(req: NextApiRequest) {
     where: {
       teamId: parentEventType.teamId,
       userId: userId,
-      OR: [{ role: "OWNER" }, { role: "ADMIN" }],
+      role: { in: ["ADMIN", "OWNER"] },
```

Comment:
Not sure why we opted for OR there in the first place, in should be a lot quicker, esp for a table of this size.